src/roslibpy_test/ros-service-call-set-bool.py

```diff
@@ -0,0 +1,18 @@
+'''
+This is the example calling services script.
+Found at: https://roslibpy.readthedocs.io/en/latest/examples.html
+'''
+
+import roslibpy
+
+client = roslibpy.Ros(host='136.25.185.6', port=9090)
```

Comment:
Don't think this script matters too much because probably won't be running the scripts in the roslibpy_test folder anymore.  Not sure whose ip this is.

Reply:
Nevermind, this is my public ip lol.